docs/send-data/opentelemetry-collector/data-source-and-configurations/collect-logs.md

```diff
@@ -0,0 +1,204 @@
+---
+id: collect-logs
+title: Collect Logs
+sidebar_label: Collect Logs
+description: Learn how to collecting logs
+---
+
+Sumo Logic Distribution for OpenTelemetry Collector provides various receivers for log collection. Full list of them is available in [our repository].
+This document explains with more details, the receivers which are the most commonly used for logs:
```

Comment:
```suggestion
The Sumo Logic Logic Distribution for OpenTelemetry Collector provides various receivers for log collection. You can find the full list in [our repository].

This document explains with more details, the receivers  most commonly used for logs:
```

Reply:
The sentence is correct, we can't use "The" because it's an article that should be used to specify a particular noun, whereas Sumo Logic is a proper noun.

Reply:
I know it can sound odd grammatically, but it's our branding wording as well as the tech industry standard. See these examples: 

- https://aws.amazon.com/blogs/opensource/launching-the-aws-distro-for-opentelemetry-developer-site-with-gatsby-and-graphql/
- https://help.sumologic.com/docs/integrations/containers-orchestration/kubernetes-control-plane/
- https://help.sumologic.com/docs/send-data/collect-from-other-data-sources/collect-metrics-telegraf/telegraf-collection-architecture/
- https://docs.appdynamics.com/appd-cloud/en/kubernetes-and-app-service-monitoring/install-kubernetes-and-app-service-monitoring/configure-appdynamics-cloud-kubernetes-collectors/appdynamics-distribution-for-opentelemetry-collector-settings

Reply:
"The" is incorrect here to use grammatically.

Reply:
While I agree with you, you'd need to take that up with our branding team. Sometimes in tech, financial, and healthcare writing, terms may not make sense grammatically, but from a legal and branding standpoint, we need to comply.

Reply:
Hey guys, while I appreciate the dedication to grammar here, you can find the full list in our repository is grammatically correct. Use of the definite article is always acceptable when used as an adjective.  http://www.butte.edu/departments/cas/tipsheets/grammar/articles.html#:~:text=The%20definite%20article%20(the)%20is,a%20noun%20takes%20no%20article. 

In this case the list you need is in the repository. I would not say List you need in repository.

Reply:
Hi @stacykor , that sentence was not the issue in question. It is "Sumo Logic Distribution for OpenTelemetry". Per guidance I've received from the OTEL team, throughout our branding, and our peers' products, it's "_The_ Sumo Logic Distribution...."





[1:40](https://sumologic.slack.com/archives/D03G7LN9G95/p1679172011823399)
Per Perk and the OTEL team, throughout our branding, and Google searches for our peers' products, it's "The Sumo Logic Distribution...."

Reply:
Yep, because it's a variant of the Otel collector so the definite article draws attention to the version. The Sumo Logic Distribution for Open Telemetry is the best. The Competitor Otel collector is not as good.  If it were a noun itself like LogReduce, the definite article would not be required. LogReduce provides.